### PR TITLE
Fix build issue

### PR DESCRIPTION
**Pull request by erilyth**
The notif and getDiff functions still give errors even when I export them, so temporarily I just ignored those 3 lines. What should be done?

Fixes T72.

### CODE REVIEW

**Comment by polybuildr** [comment]
Nice work! :D

Those give errors because they're within the scope of the functions, JSHint knows they cannot possibly be exported. :P @nisargjhaveri might be able to give better advice, but my recommendation is that you just remove the variable assignment part of the code.

Change

``` javascript
var notif = notify()...
```

to

``` javascript
notify()...
```

**Comment by polybuildr** [comment]
For `origFile`, please add a `var origFile;` statement above `setupEdit`. That will get rid of that error. 

Also add a `var` before `checkRemote`, that will fix another error.

**Comment by polybuildr** [comment]
Once those two small edits are made, we can merge this. It will be nice to have a passing build again.

File: src/static/scripts/file_edit.js [hunk 1]
```diff
@@ -10,13 +15,12 @@ function setupEdit() {
...
  */
-checkRemote = (function() {
+var checkRemote = (function() {
     var dmp;
     var failCount = 0;
     var remoteWIP = false;
     var newVersionId;
-
-    function getDiff(olddata, newdata) {
+    function getDiff(olddata, newdata) { // jshint ignore:line
```
Comment (erilyth) [inline src/static/scripts/file_edit.js:23]: What should be done about this?

Comment (polybuildr) [inline src/static/scripts/file_edit.js:23]: There are probably better ways to handle this, but for this PR, this is fine.

**Comment by erilyth** [comment]
Yea, I will remove that.

**Comment by polybuildr** [comment]
This looks great! Nice work. :D As usual, can we get a squash before merging? :P

**Comment by polybuildr** [comment]
{Fixes T72}

**Comment by erilyth** [comment]
Sure Ill squash them.

**Comment by polybuildr** [comment]
Awesome!

**Pull request merged**